Tiber upgrade (#259)
autonity/docs.autonity.org: overview/index.md

- Delegated Proof of Stake consensus for committee selection and blockchain management, using the [Tendermint BFT consensus algorithm](/concepts/consensus/pos/).
- Native [protocol assets](/concepts/protocol-assets/) and [stabilization mechanism](/concepts/asm/): _Auton_ utility coin used in the transaction fee mechanism; _Newton_ staking coin for securing the network. Auton price stability is by a CDP-based stabilization mechanism, the [Auton Stabilization Mechanism (ASM)](/concepts/asm/); the stabilization target is the [Autonomous Currency Unit (ACU)](/concepts/asm/#acu), a base-invariant, volatility-minimized index.
- [Liquid staking](/concepts/staking/#liquid-staking) for capital efficiency and a [Penalty-Absorbing Stake](/concepts/staking/#penalty-absorbing-stake-pas) model. [Delegated](/glossary/#delegated) _Newton_ stake yields transferrable _Liquid Newton_ tokens, which can be used to redeem the underlying _Newton_ by unbonding stake. [Self-bonded](/glossary/#self-bonded) stake by validator operators does not yield _Liquid Newton_ and is slashed before delegated stake to assure validator integrity and 'skin in the game'.
- Consensus accountability protocols:

>   - [Accountable fault detection (AFD)](/concepts/afd/): a protocol for detecting infractions of consensus rules by validators participating in block proposal and voting as a committee member. Failure to adhere to these rules is a rule infraction and AFD will detect and apply slashing penalties for proven rule infractions.

Reviewer: ```suggestion
  - [Accountable fault detection (AFD)](/concepts/afd/): a protocol for detecting infractions of consensus rules by committee members. Failure to adhere to these rules is a rule infraction and AFD will detect and apply slashing penalties for proven rule infractions.
```

>   - [Omission fault detection (OFD)](/concepts/ofd/): a protocol for detecting failure to vote in [consensus rounds](/concepts/consensus/pos/#consensus-round-and-internal-state) by a committee member and the failure of block proposers to provide activity proofs in block headers [outside of the protocol's delta window when they should](/concepts/ofd/#delta-delta). Failure to participate is a loss of liveness and OFD will detect and apply slashing penalties for proven inactivity.

Reviewer: ```suggestion
  - [Omission fault detection (OFD)](/concepts/ofd/): a protocol for detecting failure to vote in [consensus rounds](/concepts/consensus/pos/#consensus-round-and-internal-state) by a committee member and the failure of block proposers to provide activity proofs in block headers [when required](/concepts/ofd/#delta-delta). Failure to participate is a loss of liveness and OFD will detect and apply slashing penalties for proven inactivity.
```

> - [Oracle network](/concepts/oracle-network/): a protocol for consensus-computed median price data for selected FX currency pairs as an L1 platform feature. The submission of price data for aggregation and voting is a validator responsibility; validator nodes form an oracle network.

Reviewer: I think it could be a bit rephrased to pass the message that only active committee members are allowed to vote. 

However I do agree that all active validators (even the ones outside the committee) should have an oracle running, so that if they are selected they are ready.

Maybe it is sufficient to add something like "Only current committee members are allowed to vote".

Author: Reworded to clarify to:

>- [Oracle network](/concepts/oracle-network/): a protocol for consensus-computed weighted average price data for selected FX currency pairs as an L1 platform feature. The submission of price data for aggregation and voting is a validator responsibility. Validators form an oracle network, with each validator node running an [oracle server](/concepts/oracle-server) to source price data. Only consensus committee members participate in oracle [voting rounds](/concepts/oracle-network/#voting-rounds).

Commit ed512d525f39260dc7526c43ea2aa404965c0a3a

> - [Oracle accountability and fault detection (OAFD)](/concepts/oafd/): a protocol for detecting infractions of consensus rules by validator oracles participating in oracle voting rounds and price reporting as a committee member. Failure to adhere to these rules is a rule infraction and OAFD will detect and apply slashing penalties for proven rule infractions.

Reviewer: ```suggestion
- [Oracle accountability and fault detection (OAFD)](/concepts/oafd/): a protocol for detecting misbehaving oracles, which will be filtered via outlier detection and slashed accordingly.
```

Reviewer: I rephrased a bit, because it seemed like the OAFD had something to do with consensus rules, but it is not the case.

- [Autonity Oracle Server (AOS)](/concepts/oracle-server/) for an overview of AOS features.
- [Validator](/concepts/validator) for the role of validators and the functions they perform in securing an Autonity network.
- [Staking](/concepts/staking/) for protocol staking mechanisms and Autonity's model for liquid staking.
- Accountability protocols for validators participating in consensus and submitting price reports to the oracle protocol on-chain as committee members:
  - [Accountability fault detection protocol (AFD)](/concepts/afd/) for failing to follow consensus rules.
  - [Omission fault detection protocol (OFD)](/concepts/ofd/) for failing to participate in consensus.

>   - [Oracle accountability fault detection protocol (OAFD)](/concepts/oafd/) for failing to submit accurate price reports to the oracle protocol on-chain.

Reviewer: ```suggestion
  - [Oracle accountability fault detection protocol (OAFD)](/concepts/oafd/) for failing to submit accurate price reports to the oracle on-chain contract.
```

| [Consensus Committee](/concepts/consensus/committee/) | The subset of _validator_ nodes that participate in the consensus protocol. The Consensus Committee is updated periodically (every epoch), according to an algorithm prescribed by protocol. |
| [Oracle network](/concepts/oracle-network) | The network of validator-operated oracles that submits price data from off-chain external data providers on-chain and votes on agreeing an aggregated median price data according to an oracle protocol. |
| [Accountability fault detection (AFD)](/concepts/afd/) | A protocol for detecting infractions of consensus rules by validators participating in consensus as _consensus committee_ members. |
| [Omission fault detection protocol (OFD)](/concepts/ofd/) | A protocol for detecting inactivity and failure to vote in consensus rounds by validators participating in consensus as _consensus committee_ members. |

> | [Oracle accountability fault detection protocol (OAFD)](/concepts/oafd/) | A protocol for detecting failure to submit accurate price reports to the oracle protocol on-chain by validators participating in consensus as _consensus committee_ members. |

Reviewer: ```suggestion
| [Oracle accountability fault detection protocol (OAFD)](/concepts/oafd/) | A protocol for detecting failure to submit accurate price reports to the oracle on-chain contract by validators participating in consensus as _consensus committee_ members. |
```
